--- mmm-e41eee502568d479c6483ecae7f554312ab4ad27/.mm/bin.orig	2015-04-15 17:22:26.000000000 +0200
+++ mmm-e41eee502568d479c6483ecae7f554312ab4ad27/.mm/bin	2015-11-23 19:55:08.826073329 +0100
@@ -12,11 +12,11 @@
 BIN_LD_FLAGS += $(shell test "$(BIN_PKGMODULES)" 2>/dev/null && pkg-config $(BIN_PKGMODULES) --libs)
 
 $(BINARY): $(BIN_CFILES) $(LIBNAME_A)
-	@echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS) $(BIN_LD_FLAGS)  \
-		$(BIN_CFILES) $(LIBNAME_A) -o $@
+	echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS)  \
+		$(BIN_CFILES) $(LIBNAME_A) $(BIN_LD_FLAGS) -o $@
 
 $(BINARY).efence: $(BIN_CFILES) $(LIBNAME_A) 
-	@echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS) $(BIN_LD_FLAGS) \
-		$(BIN_CFILES) $(LIBNAME_A) -lefence -o $@
+	echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS) \
+		$(BIN_CFILES) $(LIBNAME_A) $(BIN_LD_FLAGS) -lefence -o $@
 
 all: $(BIN_TARGETS)
--- mrg-b0cb507a366775a008cf083e1bcac07623d04ae2/.mm/lib.orig	2015-11-23 20:53:44.365925795 +0100
+++ mrg-b0cb507a366775a008cf083e1bcac07623d04ae2/.mm/lib	2015-11-23 20:56:53.995917836 +0100
@@ -16,23 +16,23 @@
 LIBNAME_A=$(LIBNAME).a
 
 $(LIBNAME).a: $(LIB_OBJS)
-	@echo "  AR" $@;ar rcs $@ $(LIB_OBJS)
+	echo "  AR" $@;ar rcs $@ $(LIB_OBJS)
 
 CLEAN_FILES += symbols
 
 symbols: Makefile $(LIB_OBJS)
-	@echo " gen" $@;  nm -a $(LIB_OBJS) | grep " T " | cut -f 3 -d ' ' | grep "^$(SYMBOL_PREFIX)" | grep -v "^_" | sort > $@
+	echo " gen" $@;  nm -a $(LIB_OBJS) | grep " T " | cut -f 3 -d ' ' | grep "^$(SYMBOL_PREFIX)" | grep -v "^_" | sort > $@
 
 $(LIBNAME).$(LIBEXT): $(LIB_OBJS) symbols
-	@echo "  LD" $@;$(LD) -shared $(LIB_OBJS) -o $@ $(LIB_LD_FLAGS)
+	echo "  LD" $@;$(LD) -shared $(LIB_OBJS) -o $@ $(LIB_LD_FLAGS)
 
 # this rule is generic and could also belong with bin..
 # we try to be overly generic and throw in the cflags of both lib and bin..
 %.o: %.c *.h $(LIB_HFILES)
-	@echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
+	echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
 
 lib/%.o: lib/%.c lib/*.h $(LIB_HFILES)
-	@echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
+	echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
 
 LIBDIR ?= $(PREFIX)/lib
 #$(P64)
@@ -71,7 +71,7 @@
 EXAMPLES_LD_FLAGS += $(shell test "$(EXAMPLES_PKGMODULES)" 2>/dev/null && pkg-config $(EXAMPLES_PKGMODULES) --libs | sed 's/-Wl,--export-dynamic//' | sed 's/-pthread//')
 
 examples/%: examples/%.c $(LIBNAME).a
-	@echo "CCLD" $@; $(CC) -I.. -I. $(EXAMPLES_CFLAGS) $(LIB_CFLAGS) $(LIB_LD_FLAGS) $(EXAMPLES_LD_FLAGS) $< $(LIBNAME).a -o $@
+	echo "CCLD" $@; $(CC) -I.. -I. $(EXAMPLES_CFLAGS) $(LIB_CFLAGS) $(LIB_LD_FLAGS) $(EXAMPLES_LD_FLAGS) $< $(LIBNAME).a -o $@
 
 
 all: $(EXAMPLES_BINS)
--- mmm-e41eee502568d479c6483ecae7f554312ab4ad27/.mm/pkgconfig.orig	2015-04-15 17:22:26.000000000 +0200
+++ mmm-e41eee502568d479c6483ecae7f554312ab4ad27/.mm/pkgconfig	2015-11-23 20:21:30.166007161 +0100
@@ -6,7 +6,7 @@
 	@echo " gen" $@
 	@echo "prefix=$(PREFIX)"     > $@
 	@echo exec_prefix=\$${prefix} >> $@
-	@echo libdir=\$${prefix}/lib >> $@
+	@echo "libdir=$(LIBDIR)" >> $@
 	@echo includedir=\$${prefix}/include >> $@
 	@echo apiversion=$(API_VERSION) >> $@
 	@echo >> $@
--- mrg-b0cb507a366775a008cf083e1bcac07623d04ae2/Makefile.orig	2015-07-27 19:55:34.000000000 +0200
+++ mrg-b0cb507a366775a008cf083e1bcac07623d04ae2/Makefile	2015-11-23 21:23:14.215851520 +0100
@@ -60,7 +60,7 @@
 
 install: install-extra
 install-extra:
-	install mrg-host mrg-terminal mrg-edit mrg-browser $(DESTDIR)$(PREFIX)/bin/
+	cp -a mrg-host mrg-terminal mrg-edit mrg-browser $(DESTDIR)$(PREFIX)/bin/
 
 luajit/mrg_h.lua: lib/*.h Makefile
 	echo "local ffi = require'ffi'" > $@
