diff -burN mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361.orig/Makefile mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361/Makefile
--- mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361.orig/Makefile	2019-03-22 02:34:36.000000000 +0100
+++ mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361/Makefile	2019-04-09 07:39:01.369907820 +0200
@@ -69,7 +69,7 @@
 
 install: install-extra
 install-extra:
-	install mrg-host mrg-terminal mrg-edit mrg-browser $(DESTDIR)$(PREFIX)/bin/
+	cp -a mrg-host mrg-terminal mrg-edit mrg-browser $(DESTDIR)$(PREFIX)/bin/
 
 luajit/mrg_h.lua: lib/*.h Makefile
 	echo "local ffi = require'ffi'" > $@
diff -burN mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361.orig/.mm/bin mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361/.mm/bin
--- mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361.orig/.mm/bin	2019-03-22 02:34:36.000000000 +0100
+++ mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361/.mm/bin	2019-04-09 07:36:36.485673498 +0200
@@ -12,11 +12,11 @@
 BIN_LD_FLAGS += $(shell test "$(BIN_PKGMODULES)" 2>/dev/null && pkg-config $(BIN_PKGMODULES) --libs)
 
 $(BINARY): $(BIN_CFILES) $(LIBNAME_A)
-	@echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS)  \
+	echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS)  \
 		$(BIN_CFILES) $(LIBNAME_A) $(BIN_LD_FLAGS) -o $@
 
 $(BINARY).efence: $(BIN_CFILES) $(LIBNAME_A) 
-	@echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS)  \
+	echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS)  \
 		$(BIN_CFILES) $(LIBNAME_A) $(BIN_LD_FLAGS) -lefence -o $@
 
 all: $(BIN_TARGETS)
diff -burN mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361.orig/.mm/lib mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361/.mm/lib
--- mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361.orig/.mm/lib	2019-03-22 02:34:36.000000000 +0100
+++ mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361/.mm/lib	2019-04-09 07:38:13.704082927 +0200
@@ -16,23 +16,23 @@
 LIBNAME_A=$(LIBNAME).a
 
 $(LIBNAME).a: $(LIB_OBJS)
-	@echo "  AR" $@;ar rcs $@ $(LIB_OBJS)
+	echo "  AR" $@;ar rcs $@ $(LIB_OBJS)
 
 CLEAN_FILES += symbols
 
 symbols: Makefile $(LIB_OBJS)
-	@echo " gen" $@;  nm -a $(LIB_OBJS) | grep " T " | cut -f 3 -d ' ' | grep "^$(SYMBOL_PREFIX)" | grep -v "^_" | sort > $@
+	echo " gen" $@;  nm -a $(LIB_OBJS) | grep " T " | cut -f 3 -d ' ' | grep "^$(SYMBOL_PREFIX)" | grep -v "^_" | sort > $@
 
 $(LIBNAME).$(LIBEXT): $(LIB_OBJS) symbols
-	@echo "  LD" $@;$(CC) --shared $(LIB_OBJS) -o $@ $(LIB_LD_FLAGS)
+	echo "  LD" $@;$(CC) --shared $(LIB_OBJS) -o $@ $(LIB_LD_FLAGS)
 
 # this rule is generic and could also belong with bin..
 # we try to be overly generic and throw in the cflags of both lib and bin..
 %.o: %.c *.h $(LIB_HFILES)
-	@echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
+	echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
 
 lib/%.o: lib/%.c lib/*.h $(LIB_HFILES)
-	@echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
+	echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
 
 LIBDIR ?= $(PREFIX)/lib
 #$(P64)
@@ -71,12 +71,12 @@
 EXAMPLES_LD_FLAGS += $(shell test "$(EXAMPLES_PKGMODULES)" 2>/dev/null && pkg-config $(EXAMPLES_PKGMODULES) --libs | sed 's/-Wl,--export-dynamic//' | sed 's/-pthread//')
 
 examples/%: examples/%.c $(LIBNAME).a
-	@echo "CCLD" $@; $(CC) -I.. -I. $(EXAMPLES_CFLAGS) $(LIB_CFLAGS) $< $(LIBNAME).a $(EXAMPLES_LD_FLAGS) $(LIB_LD_FLAGS) -o $@
+	echo "CCLD" $@; $(CC) -I.. -I. $(EXAMPLES_CFLAGS) $(LIB_CFLAGS) $< $(LIBNAME).a $(EXAMPLES_LD_FLAGS) $(LIB_LD_FLAGS) -o $@
 
 
 all: $(EXAMPLES_BINS)
 CLEAN_FILES += $(EXAMPLES_BINS)
 help: _libhelp
 _libhelp:
-	@echo "$(LIBNAME).$(LIBEXT) dynamic shared library"
-	@echo "$(LIBNAME).a         static library"
+	echo "$(LIBNAME).$(LIBEXT) dynamic shared library"
+	echo "$(LIBNAME).a         static library"
diff -burN mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361.orig/.mm/pkgconfig mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361/.mm/pkgconfig
--- mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361.orig/.mm/pkgconfig	2019-03-22 02:34:36.000000000 +0100
+++ mrg-ba04b1ee9c63cbd50ea0ac9211fad881a0724361/.mm/pkgconfig	2019-04-09 07:38:42.905609331 +0200
@@ -6,7 +6,7 @@
 	@echo " gen" $@
 	@echo "prefix=$(PREFIX)"     > $@
 	@echo exec_prefix=\$${prefix} >> $@
-	@echo libdir=\$${prefix}/lib >> $@
+	@echo "libdir=$(LIBDIR)" >> $@
 	@echo includedir=\$${prefix}/include >> $@
 	@echo apiversion=$(API_VERSION) >> $@
 	@echo >> $@
